downstairs/src/region.rs

```diff
@@ -32,8 +32,30 @@ use crate::extent::{
  * There is only one repair file: the raw file itself (which also contains
  * structured context and metadata at the end).
  */
-pub fn validate_repair_files(eid: usize, files: &[String]) -> bool {
-    files == [extent_file_name(eid as u32, ExtentType::Data)]
+pub fn validate_repair_files(
```

Comment:
Nit: how about splitting this into `validate_repair_files` and `validate_replacement_files`, since they're totally independent paths (and the docstring is wrong if this function does both)?

Reply:
See what I did in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)